guides/araujo/FTP Writeup.md

```diff
@@ -0,0 +1,92 @@
+---
+
+
+---
+
+<hr>
+<h2 id="layout-posttitle-how-to-setup-an-ftp-serverauthor-joshua-araujo">layout: post<br>
+title: How to Setup an FTP Server<br>
+author: Joshua Araujo</h2>
+<h3 id="contact">Contact</h3>
+<ul>
+<li>Slack: @Shinpuru on <a href="http://wcscusf.slack.com">wcscusf.slack.com</a></li>
+<li><a href="look-at-me">LinkedIn</a></li>
+<li>Email: <a href="mailto:araujoj@mail.usf.com">araujoj@mail.usf.com</a></li>
+</ul>
+<h3 id="table">Table</h3>
+<ol>
+<li><a href="#id-link-to-section">Prerequisites</a></li>
```

Comment:
When this is merged in along with Nestor's and Sharon's guides make another pull request with the new links. For now this is good, just a note for the future.